python/py-hgexperimental/Portfile

```diff
@@ -30,6 +31,13 @@ if {${name} ne ${subport}} {
                         port:py27-cython \
                         port:lz4
 
+    pre-configure {
+        # fix error: 'register' storage class specifier is deprecated and incompatible with C++17
+        # we build with -std=c++0x and 'register' is only deprecated
+        # see https://stackoverflow.com/questions/49692794/c-17-compatability-with-python-2-7
```

Comment:
You said that you asked upstream to fix this. In that case it makes more sense to link to the upstream ticket.